test/TMxpEntityTagHandlerTest.cpp

```diff
@@ -0,0 +1,141 @@
+
```

Comment:
Like the codefactor.io bot says, please put in a Copyright header - you can find templates for this in the source code in the `./templates` folder - there are IIRC two identical files there that can be used automagically by Qt Creator when you create a `.cpp` or a `.h` file...